cylc/flow/scripts/lint.py

```diff
@@ -335,12 +438,20 @@ def check_cylc_file(dir_, file_, checks, modify=False):
     return count
 
 
-def get_cylc_files(base: Path) -> Generator[Path, None, None]:
+def get_cylc_files(
+    base: Path, exceptions: Union[list, None] = None
```

Comment:
Maybe "exclusions" instead of "exceptions" to avoid confusion with `Exception`s?

Reply:
Yes, good call. I went ahead and made it all as consistent as I could with the Flake8 config.